addons/payment_xendit/models/payment_transaction.py

```diff
@@ -17,6 +19,26 @@
 class PaymentTransaction(models.Model):
     _inherit = 'payment.transaction'
 
+    def _get_specific_processing_values(self, processing_values):
+        """ Override of payment to return Xendit-specific processing values.
+
+        Note: self.ensure_one() from `_get_processing_values`
+
+        :param dict processing_values: The generic processing values of the transaction
+        :return: The dict of provider-specific processing values
+        :rtype: dict
+        """
+        res = super()._get_specific_processing_values(processing_values)
+        if self.provider_code != 'xendit':
+            return res
+
+        rounding = const.CURRENCY_DECIMALS.get(self.currency_id.name) if self.currency_id.name in \
+            const.CURRENCY_DECIMALS else self.currency_id.decimal_places
```

Comment:
Simpler > Smart when considering clarity
```suggestion
        if self.currency_id.name in const.CURRENCY_DECIMALS:
            rounding = const.CURRENCY_DECIMALS.get(self.currency_id.name) 
        else:
            rounding = self.currency_id.decimal_places
```